From 36dab45b89a65dc2eb4a0a5efda08f6c5d2e8856 Mon Sep 17 00:00:00 2001
From: NeilBrown <neilb@suse.de>
Date: Mon, 3 Nov 2014 11:58:06 +1100
Subject: [PATCH] mdcheck: don't git error if not /dev/md?* devices exist.

If there are no such devices, the 'for' will set '$dev' to
'/dev/md?*', which should be ignored.

Signed-off-by: NeilBrown <neilb@suse.de>
---
 misc/mdcheck | 1 +
 1 file changed, 1 insertion(+)

diff --git a/misc/mdcheck b/misc/mdcheck
index 33570b9..ca5fe84 100644
--- a/misc/mdcheck
+++ b/misc/mdcheck
@@ -77,6 +77,7 @@ find /var/lib/mdcheck -name "MD_UUID*" -type f -mtime +180 -exec rm {} \;
 cnt=0
 for dev in /dev/md?*
 do
+	[ -e "$dev" ] || continue
 	sys=`sysname $dev`
 	if [ ! -f "$sys/md/sync_action" ]
 	then # cannot check this array
-- 
2.3.4

